src/responsibleai/rai_analyse/gather_rai_insights.py

```diff
@@ -75,6 +77,20 @@ def main(args):
 
         my_run = Run.get_context()
         rai_temp = create_rai_insights_from_port_path(my_run, args.constructor)
+
+        # We need to fix the following issue in RAIInsights.
+        # When using the served model wrapper, predictions (currently
+        # forecasts only) are stored as lists instead of numpy.ndarray.
```

Comment:
I believe the predictions should be numpy arrays - that is the contract we agreed on.  If there some issue with the forecasting models, then we should ideally wrap them with ml-wrappers code.

Reply:
You're absolutely right about numpy arrays.

It's not a forecasting issue really. It's an issue with the wrapper code that I wrote. I believe this here is just a list: https://github.com/microsoft/responsible-ai-toolbox/blob/4d8d9b76cd52dfa67e042caff5d864f7b23a068f/responsibleai/responsibleai/_internal/_served_model_wrapper.py#L56
Not a big deal, though. I'll fix it in a small PR in responsibleai and with the next version bump we can remove it here.